src/lib/permissions/PermissionsAPI.web.js

```diff
@@ -0,0 +1,30 @@
+class PermissionsAPIWeb {
+  async getCameraPermissionStatus() {
```

Comment:
remember, we've agreed to
1. add enum with kind of permission e.g.
```
const Permission = {
  Camera: 'camera',
  Clipboard: 'clipboard'
}
```
2. add one **public** method
```
async query(permission): Promise<boolean>
```
3. make other helper methods "private" (e.g. `_queryPermission, _checkCameraAvailability, _requestCameraAccess, _requestClipboardAccess` and add them `/** @private */` comment block)

Reply:
done